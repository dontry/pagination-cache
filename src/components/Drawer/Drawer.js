import React from "react";
import PropTypes from "prop-types";
import { makeStyles } from "@material-ui/core/styles";
import SwipeableDrawer from "@material-ui/core/SwipeableDrawer";
import { Typography } from "@material-ui/core";

const useStyles = makeStyles({
  contentWrapper: {
    width: 300
  },
  infoItemWrapper: {
    display: "flex"
  },
  infoItemLabel: {
    flexBasis: 50
  },
  infoItemValue: {
    flexGrow: 1,
    borderBottom: "1px solid #acacac"
  }
});

const ServiceDataType = {
  parent: PropTypes.string,
  made_sla: PropTypes.string,
  caused_by: PropTypes.string,
  watch_list: PropTypes.string,
  upon_reject: PropTypes.string,
  child_incidents: PropTypes.string,
  hold_reason: PropTypes.string,
  approval_history: PropTypes.string,
  resolved_by: PropTypes.string,
  sys_updated_by: PropTypes.string,
  opened_by: PropTypes.string,
  user_input: PropTypes.string,
  sys_created_on: PropTypes.string,
  sys_domain: PropTypes.string,
  state: PropTypes.string,
  sys_created_by: PropTypes.string,
  knowledge: PropTypes.string,
  order: PropTypes.string,
  calendar_stc: PropTypes.string,
  closed_at: PropTypes.string,
  cmdb_ci: PropTypes.string,
  delivery_plan: PropTypes.string,
  impact: PropTypes.string,
  active: PropTypes.string,
  work_notes_list: PropTypes.string,
  business_service: PropTypes.string,
  priority: PropTypes.string,
  sys_domain_path: PropTypes.string,
  rfc: PropTypes.string,
  time_worked: PropTypes.string,
  expected_start: PropTypes.string,
  opened_at: PropTypes.string,
  business_duration: PropTypes.string,
  group_list: PropTypes.string,
  work_end: PropTypes.string,
  caller_id: PropTypes.string,
  reopened_time: PropTypes.string,
  resolved_at: PropTypes.string,
  approval_set: PropTypes.string,
  subcategory: PropTypes.string,
  work_notes: PropTypes.string,
  close_code: PropTypes.string,
  correlation_display: PropTypes.string,
  delivery_task: PropTypes.string,
  work_start: PropTypes.string,
  additional_assignee_list: PropTypes.string,
  business_stc: PropTypes.string,
  description: PropTypes.string,
  calendar_duration: PropTypes.string,
  close_notes: PropTypes.string,
  notify: PropTypes.string,
  sys_class_name: PropTypes.string,
  closed_by: PropTypes.string,
  follow_up: PropTypes.string,
  parent_incident: PropTypes.string,
  contact_type: PropTypes.string,
  reopened_by: PropTypes.string,
  urgency: PropTypes.string,
  problem_id: PropTypes.string,
  company: PropTypes.string,
  reassignment_count: PropTypes.string,
  activity_due: PropTypes.string,
  severity: PropTypes.string,
  comments: PropTypes.string,
  approval: PropTypes.string,
  sla_due: PropTypes.string,
  comments_and_work_notes: PropTypes.string,
  due_date: PropTypes.string,
  sys_mod_count: PropTypes.string,
  reopen_count: PropTypes.string,
  sys_tags: PropTypes.string,
  escalation: PropTypes.string,
  upon_approval: PropTypes.string,
  location: PropTypes.string,
  category: PropTypes.string
};

const Drawer = ({ isOpen, close, open, number, data }) => {
  const handleOpen = () => {
    open();
  };
  const handleClose = () => {
    close();
  };

  return (
    <SwipeableDrawer
      anchor="right"
      open={isOpen}
      onOpen={handleOpen}
      onClose={handleClose}
    >
      <DrawerContent number={number} data={data} />
    </SwipeableDrawer>
  );
};

Drawer.propTyeps = {
  isOpen: PropTypes.bool.isRequired,
  close: PropTypes.func.isRequired,
  data: PropTypes.shape(ServiceDataType),
  number: PropTypes.string
};

export const DrawerContent = ({ number, data }) => {
  const classes = useStyles();
  return (
    <div className={classes.contentWrapper}>
      <Typography variant="h2">{number}</Typography>
      {Object.entries(data).map(([key, value]) => (
        <InfoItem key={key} label={key} value={value} />
      ))}
    </div>
  );
};

DrawerContent.propTypes = {
  number: PropTypes.string.isRequired,
  data: PropTypes.shape(ServiceDataType)
};

export const InfoItem = ({ label, value = "" }) => {
  const classes = useStyles();
  return (
    <div className={classes.infoItemWrapper}>
      <span className={classes.infoItemLabel}>{label}</span>
      <span className={classes.infoItemValue}>{value}</span>
    </div>
  );
};

InfoItem.propTypes = {
  label: PropTypes.string.isRequired,
  value: PropTypes.string.isRequired
};

export default Drawer;
