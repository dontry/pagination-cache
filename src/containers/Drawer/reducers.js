import { OPEN_CARD, CLOSE_CARD } from "../CardGrid/actions";
import { TOGGLE_DRAWER } from "./actions";

const initialState = {
  isOpen: false
};

export const isDrawerOpen = (
  state = initialState.isOpen,
  { type, payload }
) => {
  switch (type) {
    case OPEN_CARD:
      return true;
    case CLOSE_CARD:
      return false;
    case TOGGLE_DRAWER:
      return payload.isOpen;
    default:
      return state;
  }
};
