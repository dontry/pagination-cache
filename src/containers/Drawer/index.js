import Drawer from "../../components/Drawer";
import { selectActiveCard } from "../CardGrid/selectors";
import { connect } from "react-redux";
import { TOGGLE_DRAWER } from "./actions";
import _ from "lodash";

const mapStateToProps = state => {
  const activeCard = selectActiveCard(state) || {};
  return {
    data: activeCard.serviceData || {},
    number: _.get(activeCard, "coreData.number", "xx"),
    isOpen: state.ui.isDrawerOpen
  };
};

const mapDispatchToProps = dispatch => ({
  toggle: isOpen => dispatch({ type: TOGGLE_DRAWER, payload: { isOpen } })
});

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(Drawer);
