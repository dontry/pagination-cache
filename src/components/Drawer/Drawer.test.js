import React from "react";
import { render } from "@testing-library/react";
import Drawer, { DrawerContent } from "./Drawer";

const number = "INCXXXX";

const serviceData = {
  parent: "",
  made_sla: "false",
  caused_by: "",
  watch_list: "",
  upon_reject: "null",
  child_incidents: "",
  hold_reason: "",
  approval_history: "",
  resolved_by: "System Administrator",
  sys_updated_by: "system",
  opened_by: "System Administrator",
  user_input: "",
  sys_created_on: "2017-07-10 15:14:01",
  sys_domain: "global",
  state: "toggled",
  sys_created_by: "admin",
  knowledge: "false",
  order: "",
  calendar_stc: "7,690,046",
  toggled_at: "2019-05-27 12:56:17",
  cmdb_ci: "ApplicationServerPeopleSoft",
  delivery_plan: "",
  impact: "1 - High",
  active: "false",
  work_notes_list: "",
  business_service: "",
  priority: "1 - Critical",
  sys_domain_path: "/",
  rfc: "",
  time_worked: "",
  expected_start: "",
  opened_at: "2018-12-01 15:06:52",
  business_duration: "21 Days 15 Hours 14 Minutes",
  group_list: "",
  work_end: "",
  caller_id: "Alejandro Mascall",
  reopened_time: "",
  resolved_at: "2019-02-28 15:14:18",
  approval_set: "",
  subcategory: "null",
  work_notes:
    "2019-02-28 13:17:01 - System Administrator (Work notes)\ntest\n\n",
  toggle_code: "toggled/Resolved by Caller",
  correlation_display: "",
  delivery_task: "",
  work_start: "",
  additional_assignee_list: "",
  business_stc: "1,869,258",
  description:
    "Watcher daemon detected that the CPU was 100% busy for more than 10 minutes",
  calendar_duration: "89 Days 7 Minutes",
  toggle_notes: "Fixed",
  notify: "Do Not Notify",
  sys_class_name: "Incident",
  toggled_by: "System Administrator",
  follow_up: "",
  parent_incident: "",
  contact_type: "Email",
  reopened_by: "",
  urgency: "1 - High",
  problem_id: "",
  company: "ACME Germany",
  reassignment_count: "1",
  activity_due: "2019-02-28 15:17:01",
  severity: "1 - High",
  comments: "",
  approval: "null",
  sla_due: "UNKNOWN",
  comments_and_work_notes:
    "2019-02-28 13:17:01 - System Administrator (Work notes)\ntest\n\n",
  due_date: "",
  sys_mod_count: "15",
  reopen_count: "",
  sys_tags: "",
  escalation: "Overdue",
  upon_approval: "null",
  location: "Bockenheimer LandstraÃŸe 223, Frankfurt",
  category: "Hardware"
};

describe("Drawer Component", () => {
  it("renders Drawer with service data", () => {
    const toggle = jest.fn();
    const { container } = render(
      <Drawer
        isOpen={true}
        toggle={toggle}
        number={number}
        data={serviceData}
      />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});

describe("DrawerContent Component", () => {
  it("renders Drawer with service data", () => {
    const { container, queryByText } = render(
      <DrawerContent isOpen={true} number={number} data={serviceData} />
    );
    expect(queryByText(number)).toBeTruthy();
    Object.keys(serviceData).forEach(key => {
      expect(queryByText(key)).toBeTruthy();
    });
    expect(container.firstChild).toMatchSnapshot();
  });
});
