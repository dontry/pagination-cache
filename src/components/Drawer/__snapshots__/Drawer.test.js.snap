// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Drawer Component renders Drawer with service data 1`] = `
<div
  class="PrivateSwipeArea-root-76 PrivateSwipeArea-anchorRight-78"
  style="width: 20px;"
/>
`;

exports[`DrawerContent Component renders Drawer with service data 1`] = `
<div
  class="makeStyles-contentWrapper-81"
>
  <h2
    class="MuiTypography-root MuiTypography-h2"
  >
    INCXXXX
  </h2>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      parent
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      made_sla
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      false
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      caused_by
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      watch_list
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      upon_reject
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      null
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      child_incidents
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      hold_reason
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      approval_history
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      resolved_by
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      System Administrator
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      sys_updated_by
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      system
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      opened_by
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      System Administrator
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      user_input
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      sys_created_on
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      2017-07-10 15:14:01
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      sys_domain
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      global
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      state
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      Closed
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      sys_created_by
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      admin
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      knowledge
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      false
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      order
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      calendar_stc
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      7,690,046
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      closed_at
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      2019-05-27 12:56:17
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      cmdb_ci
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      ApplicationServerPeopleSoft
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      delivery_plan
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      impact
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      1 - High
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      active
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      false
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      work_notes_list
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      business_service
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      priority
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      1 - Critical
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      sys_domain_path
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      /
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      rfc
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      time_worked
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      expected_start
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      opened_at
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      2018-12-01 15:06:52
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      business_duration
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      21 Days 15 Hours 14 Minutes
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      group_list
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      work_end
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      caller_id
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      Alejandro Mascall
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      reopened_time
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      resolved_at
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      2019-02-28 15:14:18
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      approval_set
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      subcategory
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      null
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      work_notes
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      2019-02-28 13:17:01 - System Administrator (Work notes)
test


    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      close_code
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      Closed/Resolved by Caller
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      correlation_display
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      delivery_task
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      work_start
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      additional_assignee_list
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      business_stc
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      1,869,258
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      description
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      Watcher daemon detected that the CPU was 100% busy for more than 10 minutes
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      calendar_duration
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      89 Days 7 Minutes
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      close_notes
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      Fixed
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      notify
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      Do Not Notify
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      sys_class_name
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      Incident
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      closed_by
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      System Administrator
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      follow_up
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      parent_incident
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      contact_type
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      Email
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      reopened_by
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      urgency
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      1 - High
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      problem_id
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      company
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      ACME Germany
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      reassignment_count
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      1
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      activity_due
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      2019-02-28 15:17:01
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      severity
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      1 - High
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      comments
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      approval
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      null
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      sla_due
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      UNKNOWN
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      comments_and_work_notes
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      2019-02-28 13:17:01 - System Administrator (Work notes)
test


    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      due_date
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      sys_mod_count
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      15
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      reopen_count
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      sys_tags
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    />
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      escalation
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      Overdue
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      upon_approval
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      null
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      location
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      Bockenheimer Landstraße 223, Frankfurt
    </span>
  </div>
  <div
    class="makeStyles-infoItemWrapper-82"
  >
    <span
      class="makeStyles-infoItemLabel-83"
    >
      category
    </span>
    <span
      class="makeStyles-infoItemValue-84"
    >
      Hardware
    </span>
  </div>
</div>
`;
